dynamiq/nodes/writers/milvus.py

```diff
@@ -37,13 +39,26 @@ def __init__(self, **kwargs):
             kwargs["connection"] = Milvus()
         super().__init__(**kwargs)
 
+    @model_validator(mode="after")
+    def check_required_params(self) -> "MilvusDocumentWriter":
```

Comment:
please use https://docs.pydantic.dev/2.3/api/types/#pydantic.types.PositiveInt for strict positive int validation instead of separate method